info/v1/machine.go

```diff
@@ -176,6 +176,9 @@ type MachineInfo struct {
 	// Memory capacity and number of DIMMs by memory type
 	MemoryByType map[string]*MemoryInfo `json:"memory_by_type"`
 
+	// Average power budget for NVM devices configured in BIOS.
+	NVMAvgPowerBudget uint `json:"nvm_avg_power_budget"`
```

Comment:
Is this the only NVM-related thing we plan to have here?  If not, it would be nice to have a NVMInfo struct to group them.

Reply:
There will be more. I'm going to update it then :)